Server/JokesOnYou.Web.Api/Repositories/JokesRepository.cs

```diff
@@ -34,5 +35,11 @@ public Task<JokeReplyDto> GetJokeDtoAsync(int id)
             return _context.Jokes.ProjectTo<JokeReplyDto>(_mapper.ConfigurationProvider)
                 .FirstOrDefaultAsync(j => j.Id == id);
         }
+
+        public async Task<Joke> GetJokeById(int id)
+        {
+            var joke = await _context.Jokes.FirstOrDefaultAsync(x => x.Id == id);
```

Comment:
We can return the joke directly here. Subjective thought but its good practice to follow the same pattern as the other methods, and we do save some memory.